src/binary-encoding.adoc

```diff
@@ -1096,27 +1217,49 @@ Unlike the short-form encoding, the long-form encoding reserves:
 * *12 bits for the offset*, which counts the number of minutes (not quarter-hours) from -1440
 (that is: `-24:00`). An offset value of `0b111111111111` indicates an unknown offset.
 
-If the timestamp's length is greater than or equal to `8`, it has fractional seconds that are encoded as a
-`(coefficient, exponent)` pair, similar to a <<decimals, decimal>>. However, it is illegal for the fractional
-seconds value to be greater than or equal to `1.0` or less than `0.0`. For this reason, both the coefficient and
-the exponent are encoded using unsigned types. The included coefficient `FlexUInt` is unsigned to prevent the encoding of
-fractional seconds less than `0.0`. The exponent `FixedUInt` is implicitly negative, discouraging the encoding of
-decimal numbers greater than `1.0`. Note that validation is still required; namely:
+Similar to short-form timestamps, with the exception of representing the fractional seconds, the components of the
+timestamp are encoded as bit-fields on a `FixedUInt` that corresponds to the length that followed the opcode.
+
+If the timestamp's overall length is greater than or equal to `8`, the `FixedUInt` part of the timestamp is `8` bytes
```

Comment:
```suggestion
If the timestamp's overall length is greater than or equal to `8`, the `FixedUInt` part of the timestamp is `7` bytes
```

I think this is what you're trying to say? It takes 7 bytes to encode a long-form timestamp through seconds, then the FlexUInt + FixedUInt.

Reply:
Great catch!